src/main/java/au/com/dmg/fusion/data/CustomFieldType.java

```diff
@@ -0,0 +1,9 @@
+package au.com.dmg.fusion.data;
+
+public enum CustomFieldType {
```

Comment:
Please add _Unknown_ so it can be used when handling invalid data types sent through JSON.  The data type _Unknown_ will be used by: src\main\java\au\com\dmg\fusion\Message.java (au/com/dmg/fusion/Message.java)